content/02.main-text.md

```diff
@@ -19,3 +19,23 @@ We wrote entirely in the open without restrictions on who was welcome to contrib
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
+
+
+### Authorship
+
+To determine authorship we followed the International Committee of Medical Journal Editors (ICMJE) guidelines and used GitHub to track contributions.
+ICMJE recommends authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
+We acknowledged other contributors who did not meet all four criteria, including contributors who provided text but did not review and approve the complete manuscript.
+`TODO: In Discussion, mention the challenge of determining contributions of potential authors who worked with someone else on the text but did not make their own commits`
```

Comment:
It's important that maintainers set the standard that authors should commit their own work. If the author is git-illiterate then the maintainers can make the commit and override the author info. It's important to avoid situations where say a graduate student writes the content, but their PI acts as a conduit, thereby taking the credit for the work. The nice thing about entirely open manuscripts is that we can mostly eradicate abusive credit practices in science.